wagtail/admin/views/reports/audit_logging.py

```diff
@@ -88,7 +86,11 @@ def get_queryset(self):
                 PageLogEntry.objects.filter(deleted=True).values('page_id')
             ))
 
-        return PageLogEntry.objects.filter(q)
+        # Using prefech_related() on page, as select_related() generates an INNER JOIN,
+        # which filters out entries for deleted pages
+        return PageLogEntry.objects.filter(q).select_related(
+            'user', 'user__wagtail_userprofile'
+        ).prefetch_related('page')
```

Comment:
[Looks like](https://github.com/wagtail/wagtail/blob/main/wagtail/core/models/__init__.py#L3944) `page` is a ForeignKey, so you should be able to `select_related` it here instead, resulting in 1 fewer query